examples/bank_account/bank_account.py

```diff
@@ -16,7 +16,7 @@ def info(self):
     def deposit(self, amount):
         """ deposit money """
         if amount > 0:
-            self._balance += amount - self._calc_commission_rate(self._hasCommissionDiscount)
+            self._balance -= amount - self._calc_commission_rate(self._hasCommissionDiscount)
```

Comment:
Consider adding a check to ensure that the commission rate does not exceed the deposit or withdrawal amount. [important]